### ISSUE TITLE
[FeatureRequest] Refactor menus to GAction

### ISSUE THREAD

**Issue by rilian-la-te** [mate-desktop/mate-panel#862]
#### Expected behaviour
Use GAction to back a menus (gnome-panel already done this)

#### Actual behaviour
Deprecated GtkAction is used

#### Steps to reproduce the behaviour
Install MATE

#### MATE general version
master

#### Package version
master

#### Linux Distribution
ArchLinux

#### Link to downstream report of your Distribution
Nowhere, it is a FR)

**Comment by lukefromdc** [comment]
Let's keep in mind https://github.com/mate-desktop/marco/pull/422#issuecomment-416769756 which gives a way to keep menu icons, tooltips, and drang and drop ability while using GAction instead of the older and deprecated GtkAction.

**Comment by rilian-la-te** [comment]
Yes, but that code requires internal knowledge about menu structure. If GTK decides to change it, we need to change this code.

**Comment by lukefromdc** [comment]
This would be required if we ever port to GTK 4, the reason we have not done so in the past is that without something to reimplement icons in menus we would lose those icons and that is not something we are willing to give up. Instead we have sought to make them respect the global org>mate>interface>menus-have-icons gsettings preference, which defaults to TRUE
